public/app/plugins/panel/graph/graph_tooltip.ts

```diff
@@ -172,6 +172,16 @@ export default function GraphTooltip(this: any, elem: any, dashboard: any, scope
     appEvents.emit(CoreEvents.graphClicked, { pos: pos, panel: panel, item: item });
   });
 
+  elem.bind('plotleave', () => {
+    if (panel.tooltip.shared) {
```

Comment:
Nit: maybe use quick exit?
```suggestion
    if (!panel.tooltip.shared) { return; }
```